Application/EdFi.Ods.Api.IntegrationTestHarness/UpdateAdminDatabaseTask.cs

```diff
@@ -260,11 +260,11 @@ private PostmanEnvironment UpdateAdminDatabase(string tenantIdentifier = null)
                     {
                         var key = !string.IsNullOrEmpty(client.Key)
                             ? client.Key
-                            : GetGuid();
+                            : client.ApiClientName;
```

Comment:
Don't generate random keys/secrets with every launch of the Integration Test Harness project/process anymore.